karma.conf.coffee

```diff
@@ -7,35 +7,62 @@ module.exports = (config) ->
     frameworks: [
       'mocha',
       'sinon-chai'
+      'browserify'
+      'commonjs'
     ]
 
     # list of files / patterns to load in the browser
     files: [
       # vendor
-      'node_modules/angular/angular.js',
-      'node_modules/angular-mocks/angular-mocks.js',
+      'node_modules/angular/angular.js'
+      'node_modules/angular-mocks/angular-mocks.js'
 
-      # test
-      'test/mocks.coffee',
-      'test/**/*Spec.coffee'
+      # mocks
+      'test/mocks.coffee'
 
       # client
-      'dist/angular-jitsi-meet.js',
+      'src/**/*.coffee'
+
+      # test
+      'test/**/*Spec.coffee'
     ]
 
     # list of files to exclude
     exclude: [
+      'src/ModuleDefinitions.coffee'
     ]
 
     # preprocess matching files before serving them to the browser
     # available preprocessors: https://npmjs.org/browse/keyword/karma-preprocessor
     preprocessors:
-      '**/*.coffee': ['coffee']
-
+      'src/!(index).coffee': ['coffee', 'coverage', 'commonjs']
```

Comment:
quite the array of preprocessors!  perhaps a comment or two explaining the need for the various combinations here would be helpful.